src/Libraries/CoreNodeModels/Input/ColorPalette.cs

```diff
@@ -46,6 +46,15 @@ public DSColor DsColor
             }
         }
 
+        //override ExtensionNode NodeType from NodeModel base class
+        public override string NodeType
+        {
+            get
+            {
+                return "ColorInputNode";
+            }
+        }
+
         public override NodeInputData InputData
```

Comment:
@mjkkirschner this is the only dynamo change I have made because I previously merged [this PR](url) with the json constructors

Reply:
that link doesn't go anywhere :(

Reply:
@mjkkirschner ctrl+v vs cmd+v =/
https://github.com/DynamoDS/Dynamo/pull/8226

Reply:
@alfarok are there tests in Dynamo to verify this node saves and loads correctly from JSON?

Reply:
https://github.com/DynamoDS/Dynamo/blob/dec6240ded0c4369617775336b9af60c2aba4103/test/DynamoCoreWpfTests/ColorPaletteTests.cs

Reply:
Are these for JSON? I think they are for XML, can you add some specifically for json files?

Reply:
Those are the only existing color palette tests which could be modified to check JSON as well.  One thing I was unsure about is how many new tests should be written to check XML and JSON if next sprint we are trying to run all existing XML tests against JSON.

Reply:
I think it's worth adding one or two sanity tests to make sure the work you are doing here functions and other changes do not make it regress.

Reply:
I agree, I will do something similar to the first existing test but:
- create w/s
- run 
- save
- close
- open
- run
- assert same results

Reply:
@mjkkirschner also posted an example of the node serialization in the description